Documentation/watch-recovery.md

```diff
@@ -0,0 +1,81 @@
+# Break system locally and watch recovery
+
+First, make sure you are running a sample app. Follow the [Tectonic Sandbox tutorials][first-app] to launch a simple app, or use one of your own. Once an app is running, break the cluster and watch how it recovers.
+
+One of the core features of Kubernetes is that it’s designed to maintain the desired state defined by operators and app admins. Now that we have an app running, let’s see what happen when we intentionally kill a copy of the API server, which is the brains of the cluster. The API server is run as a Deployment with multiple Pods, to ensure High Availability. Each one of these runs on a master node.
```

Comment:
Kind of a nit, but it seems like a transition sentence is needed before the third sentence. Alternatively, I think restructuring the paragraph (so that the explanation of the API server and how it acts as the brains to the cluster comes first) may read better. E.g., something like:
> The API server acts as the brains of the cluster and is run as a Deployment with multiple Pods to ensure High Availability. Each one of these runs on a master node. Now that we have an app running, let's see what happens when we intentionally kill a copy of the API server.

I think if we end with "Let's do the thing", it more naturally leads into the next section.

But if you prefer it as is, that's cool too. In the second sentence, "happen" should be changed to "happens" though.